<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordView</class>
 <widget class="QDialog" name="RecordView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>811</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1号房间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2号房间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3号房间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4号房间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5号房间：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
